solidity/hardhat.config.ts

```diff
@@ -101,6 +103,7 @@ const config: HardhatUserConfig = {
     disambiguatePaths: false,
     runOnCompile: true,
     strict: true,
+    except: ["BridgeStub$", "Bridge$"],
```

Comment:
I think we can remove that exception because of changes made in https://github.com/keep-network/tbtc-v2/pull/166

Reply:
I was able to remove `Bridge` from the list of exceptions in 6a72e290c476b8798964531fa173a184f8050c05.
Note that we will still need to remove:
```
      allowUnlimitedContractSize: true,
      blockGasLimit: 30000000,
 ```
 
Without it the Bridge does not deploy.